data/templates/cloud-config.yaml

```diff
@@ -0,0 +1,35 @@
+#cloud-config
+write_files:
+  - path: /etc/rc.local
+    permissions: "0755"
+    owner: root
+    content: |
+      #!/bin/bash
+      modprobe ipmi_devintf
+      wget -O /tmp/micro.tar.xz <%= api.server %>/micro.tar.xz
+      while [ $(docker images | grep -c micro) == "0" ]; do
+         xz -cd /tmp/micro.tar.xz | docker load
+      done
+      docker run -it -d \
```

Comment:
As I mentioned earlier, would be awesome to do this in a loop so we can change what containers are downloaded and run dynamically by the workflow, but seems more appropriate for follow-on work than necessary for this PR.